playbooks/pipeline-sample-boot-verify.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("no") }}'
+  become_user: '{{ become_user | default("root") }}'
+  remote_user: '{{ remote_user | default("root") }}'
+  tasks:
+      - set_fact:
+          real_ansible_host: "{{ ansible_host | default(inventory_hostname) }}"
+
+      - name: 01 Get rpm-ostree status --json output
+        command: rpm-ostree status --json
+        register: ros
+
+      - name: 02 Convert to JSON
```

Comment:
I think it would make the most sense to do some sort of comparison like task 12 in the ostree-boot-verify playbook, or remove some of these set_facts. I guess I don't see the gain from setting facts like ros_booted and ros_not_booted if they are never used; unless just printing them out is valuable? Maybe add a debug?

Reply:
I took this from what Bill had and just removed some steps.  I will make this playbook more generic then for all cloud images